packages/core/bootstrap/src/index.ts

```diff
@@ -142,4 +150,17 @@ export const expose = <C extends Config>(
   }
 }
 
-export { Requester, Validator, AdapterError, Builder, Logger, util, server, Cache, RateLimit }
+export {
+  Requester,
+  Validator,
+  Overrider,
+  AdapterError,
+  Builder,
+  Logger,
+  util,
+  server,
+  Cache,
+  RateLimit,
+}
+
+export type { OverrideObj }
```

Comment:
Let's keep this type either bundled with the module (since it is a * import) or move it to the repo level types.